src/components/Form.js

```diff
@@ -151,6 +151,7 @@ export default class Form extends Component {
 
   onSubmit = event => {
     event.preventDefault();
+    event.persist();
```

Comment:
Would we need to ever access the event asynchronously? Do you have a sample use case for that?

Reply:
If you are going to pass the original event past this function you need to persist or you get react warnings.